apisix/cli/apisix.lua

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env /usr/local/Cellar/openresty/1.19.3.1_1/luajit/bin/luajit
```

Comment:
@fukiki  `/usr/local/Cellar/openresty/1.19.3.1_1/luajit/bin/luajit` limits the version of OpenResty. Should we make it a conditional variable? Or just revert this if no used.

Reply:
And the path is a classical brew path, what if people run APISIX on Linux distro?

Reply:
A new shell script `apisix` will to call `luajit bin/apisix.lua`, and luajit path will be captured dynamically.
The `#!/usr/bin/env` will be delete in `apisix/cli/apisix.lua`